### PR TITLE
64 restrict airbyte destinations to bigquery and postgres

### PR DESCRIPTION
**Pull request by fatchat**
if `AIRBYTE_DESTINATION_TYPES` exists in `.env`, treat it as a comma-separated string of allowed destinations

e.g.
    `AIRBYTE_DESTINATION_TYPES="Postgres,BigQuery"`

### CODE REVIEW

File: ddpui/api/client/airbyte_api.py [hunk 1]
```diff
@@ -268,6 +269,9 @@ def get_airbyte_destination_definitions(request):
         raise HttpError(400, "create an airbyte workspace first")
 
     res = airbyte_service.get_destination_definitions(orguser.org.airbyte_workspace_id)
```
Comment (siddhant3030) [inline ddpui/api/client/airbyte_api.py:271]: why don't you use this directly?  org = request.orguser.org and then use org everywhere

File: ddpui/api/client/airbyte_api.py [hunk 2]
```diff
@@ -268,6 +269,9 @@ def get_airbyte_destination_definitions(request):
...
+    allowed_destinations = os.getenv('AIRBYTE_DESTINATION_TYPES')
+    if allowed_destinations:
+        res = [destdef for destdef in res if destdef['name'] in allowed_destinations.split(',')]
```
Comment (siddhant3030) [inline ddpui/api/client/airbyte_api.py:274]: I would write this something like this 

if allowed_destinations:
        destination_definitions = [d for d in destination_definitions if d['name'] in allowed_destinations.split(',')] 
        
and we can change the above res to destination_definitions

File: ddpui/api/client/airbyte_api.py (hunk 1 above)
Comment (fatchat) [reply ddpui/api/client/airbyte_api.py:271]: i could but it hardly makes a difference... readability is the same, i doubt there is a performance difference and even if there is this is not meant to be high-performant code. it's the kind of change that if someone else did it i would roll my eyes but not stop them

File: ddpui/api/client/airbyte_api.py (hunk 2 above)
Comment (fatchat) [reply ddpui/api/client/airbyte_api.py:274]: so you would rename `res` to `destination_definitions`? you could... all the other functions have `res` though

`res` is not as meaningful i agree, but if you change it in this function you should change it to more meaningful names everywhere else in the file

Comment (siddhant3030) [reply ddpui/api/client/airbyte_api.py:274]: Yes ideally we should do that but not on priority right now. I'll try to get involved in the PR review more often now.

**Review by Ishankoradia — APPROVED** [review]

**Pull request merged**